db/seeds/activities/stem_learning.rb

```diff
@@ -1946,3 +1946,36 @@
 end
 
 a.programmes << secondary_certificate unless a.programmes.include?(secondary_certificate)
+
+########################################################################################################################
```

Comment:
It doesn't seem like we have banner comments in the rest of the file, could we not have them here please